metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/indexbuilder/ESIndexBuilder.java

```diff
@@ -53,6 +55,7 @@ public void buildIndex(String indexName, Map<String, Object> mappings, Map<Strin
     Map<String, Object> baseSettings = new HashMap<>(settings);
     baseSettings.put("number_of_shards", numShards);
     baseSettings.put("number_of_replicas", numReplicas);
+    baseSettings.put("refresh_interval", String.format("%ss", refreshIntervalSeconds));
```

Comment:
Why are we doing a String.format here? Does Elastic need it to be a String?

Reply:
Yes, requires a string with the unit 's' for seconds.